ansible/roles_ocp_workloads/ocp4_workload_rhte19_shadowman/tasks/workload.yml

```diff
@@ -77,7 +79,10 @@
   delay: 10
 
 - set_fact:
-    datagrid_developer_password: "{{ infinispan_creds.resources[0].data['identities.yaml'] | b64decode | from_yaml | json_query(\"credentials[?(@.username=='developer')].password\") | join }}"
+    datagrid_developer_password: >-
+      {{ infinispan_creds.resources[0].data['identities.yaml']
+      | b64decode | from_yaml
+      | json_query(\"credentials[?(@.username=='developer')].password\") | join }}
```

Comment:
Error here, note that the `\` is not needed once the quote operator is changed. Check with yaml to json conversion...

Reply:
Will check.